06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # this is an example of a fairly skewed distribution (BRCA has a large LOF)
</pre><p>Not sure if I follow here. You mention BRCA but other genes are plotted below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>